docs/book/src/quick-start.md

```diff
@@ -7,9 +7,19 @@ This Quick Start guide will cover:
 - [Running locally](#test-it-out-locally)
 - [Running in-cluster](#run-it-on-the-cluster)
 
+## Prerequisites
```

Comment:
maybe just merge this with installation?  `kubebuilder` isn't exactly a prerequisite, since we show how to install it below.

Reply:
Make sense remove kubebuilder

Reply:
The kubebuilder makes sense not be in the prerequisite,  however, the others one shows that should be.